### PR TITLE
Adding a "quote" section on the Markdown formatting page

### PR DESCRIPTION
**Pull request by Globulle**
I've added a small section for the quote syntax, using `>`.

(as it's my first use of Github, I hope I've done it right !)

### CODE REVIEW

File: app/views/pages/formatting.html.erb [hunk 1]
```diff
@@ -49,6 +49,13 @@
 2. <%= t '.lists_code3' %>
 3. <%= t '.lists_code4' %></pre>
 
+      <h3><%= t '.quote' %></h3>
+      <p><%= t '.quote_ex1' %></p>
+      <pre><%= t '.quote_code1' %>
+&gt; <code><%= t '.quote_code2' %></code>
```
Comment (jhass) [inline app/views/pages/formatting.html.erb]: Isn't the `&gt;` redundant?

Comment (Globulle) [inline app/views/pages/formatting.html.erb]: I was expecting the `>`  character to be interpreted by HTML so I used its raw code : `&gt;`. I must admit I've not been able to test whether the rendering is correct though.

Comment (Globulle) [inline app/views/pages/formatting.html.erb]: Oops sorry you're right. It has to be removed from `en.yml` file

Comment (jhass) [inline app/views/pages/formatting.html.erb]: If you remove it from en.yml, it needs to be inside the code tag, no?

Comment (Globulle) [inline app/views/pages/formatting.html.erb]: I don't think so because it is not the case for the list examples above... But I may be wrong.

Comment (jhass) [inline app/views/pages/formatting.html.erb]: The entire list example is in a `<pre>` tag

Comment (Globulle) [inline app/views/pages/formatting.html.erb]: This is also the case for the quote example.

Comment (jhass) [inline app/views/pages/formatting.html.erb]: oh, you're right, I'm sorry.

Comment (Globulle) [inline app/views/pages/formatting.html.erb]: No pb. :-) By the way, how can we preview such a page in order to check the code before committing changes? That's not obvious to me...

File: config/locales/en.yml [hunk 2]
```diff
@@ -526,6 +526,11 @@ en:
       lists_code4: "item three."
       lists_code5: "This will become another bulleted list:"
       lists_code6: "And this will be a numbered list:"
+      quote: "Quotation"
+      quote_ex1: "When you want to quote an extract of an article or another comment, you can conveniently format your text by starting the line or the  paragraph by the “>”" character:"
```
Comment (jhass) [inline config/locales/en.yml]: There's an extra " here that breaks parsing the file.

File: app/views/pages/formatting.html.erb (hunk 1 above)
Comment (jhass) [inline app/views/pages/formatting.html.erb]: Here's how it looks like currently.

File: config/locales/en.yml (hunk 2 above)
Comment (Globulle) [inline config/locales/en.yml]: Ok, thanks. I should have seen it, sorry.

**Comment by jhass** [comment]
Alright, thanks!

**Comment by Flaburgan** [comment]
Thank you Globulle! Congratulation for your first contribution!

**Comment by denschub** [comment]
pushed to production.

**Pull request merged**